docker/checkout.bash

```diff
@@ -63,3 +66,19 @@ cd ibeo_msgs
 echo "" > COLCON_IGNORE
 cd ../astuff_sensor_msgs
 echo "" > COLCON_IGNORE
+
+cd ../
+
+#rosbridge_suite is a ROS meta-package including all the rosbridge packages.
+git clone https://github.com/usdot-fhwa-stol/rosbridge_suite -b ros2
+
+
+cd ${dir}/src
+
+# git clone --branch master --depth 1 https://github.com/nitroshare/qhttpengine.git
+git clone -b master --depth 1 https://github.com/etherealjoy/qhttpengine.git
+git clone -b isetta --depth 1 https://github.com/usdot-fhwa-OPS/V2X-Hub.git
```

Comment:
Need to update to clone final V2X-Hub tag

Reply:
@JonSmet whey do we have a git clone for V2XHub.git here? How does this relate to platform code?

Reply:
It looks like this was added in December 2022 as part of PR #1981 in support of the CARMA Cloud Client (IHP2) development. Specifically, it enabled CARMA Platform to include V2XHub's ccserver package in its build.

Reply:
@saina-ramyar  please re-confirm this is needed and which branch it should be pointing to. I assume master? . @SaikrishnaBairamoni once confirmed, I think you can update this file for @codygarver to merge.

Reply:
it will need to be changed to whatever tag we end up using in V2X-Hub, `something-something-7.4.0` presumably

Reply:
Yes, it is needed for the cellular communication. it should be the main branch, so "develop"